trip-to-carbon.ts

```diff
@@ -1,8 +1,9 @@
-import axios from 'axios';
+import axios from 'axios'
```

Comment:
Why did this line, and many others in this file, change?

Reply:
I'm not actually sure, I have fixed the weird spacing issues.